test/plugin.html-validate.external-links.mjs

```diff
@@ -149,52 +175,51 @@ export default class ExternalLinksRule extends Rule {
     }
   }
 
-  // Check for href external links
-  tagReady({ target }) {
-    // TODO: also check image.src, link.href, script.src
-    if (target.tagName !== "a") {
-      return;
-    }
-
-    if (!target.hasAttribute("href")) {
-      return;
-    }
-
-    // Decode the URL from the href attribute, see https://gitlab.com/html-validate/html-validate/-/issues/218
-    // Quickly replace a few common HTML entities, TODO use a real approach for this
-    const url = target.getAttribute("href").value.replace(/&amp;/g, "&").replace(/&gt;/g, ">").replace(/&lt;/g, "<");
-
-    if (/^https?:\/\//i.test(url) === false) {
-      return;
-    }
+  domReady({ document }) {
```

Comment:
This removed URL decoding. URL decoding is good because:

`<a href="https://example.com/a&amp;b">` is for the url `https://example.com/a&b`